tests/baselines/reference/organizeImports/UnusedTrivia2.ts

```diff
@@ -6,6 +6,6 @@ F1();
 
 // ==ORGANIZED==
 
-/*A*/ import /*B*/ { /*C*/ F1 /*D*/ } /*G*/ from /*H*/ "lib" /*I*/; /*J*/ //K
```

Comment:
I'm not sure why we lost the space, but it doesn't seem very problematic since this commenting pattern is very unusual.

Reply:
It looks like the space wasn't present in the input, so this is a good change.